lib/extend/tag.js

```diff
@@ -228,15 +226,8 @@ class Tag {
       options = {};
     }
 
-    const cache = [];
-
-    const escapeContent = str => `<!--${placeholder}${cache.push(str) - 1}-->`;
-
-    str = str.replace(/<pre><code.*>[\s\S]*?<\/code><\/pre>/gm, escapeContent);
```

Comment:
Hexo only use `tag.render` in `post.js`, which is after `backtick_code_filter` is executed.

In `backtick_code_filter`, `{` & `}` are already escaped and it is fine to render with nunjucks directly.